ui/src/state/groups/groups.ts

```diff
@@ -76,21 +80,35 @@ export function useGroups() {
 }
 
 export function useGroup(flag: string, withMembers = false, subscribe = false) {
+  const queryClient = useQueryClient();
   const initialData = useGroups();
   const group = initialData?.[flag];
-  const { data, ...rest } = useReactQuerySubscription({
-    queryKey: ['groups', flag],
+  const queryKey = useMemo(() => ['groups', flag], [flag]);
+
+  const { data, ...rest } = useReactQueryScry({
+    queryKey,
     app: 'groups',
-    path: `/groups/${flag}/ui`,
-    initialScryPath: `/groups/${flag}`,
-    enabled: !!flag && flag !== '' && withMembers && subscribe,
-    initialData: group,
+    path: `/groups/${flag}`,
     options: {
+      enabled: !!flag && flag !== '' && withMembers && subscribe,
```

Comment:
I missed this before, but we can't just set withMembers to be true, we must always set both withMembers and subscribe?